package.json

```diff
@@ -343,6 +351,7 @@
     "s3rver": "github:20minutes/s3rver",
     "sass": "^1.66.1",
     "sass-loader": "^13.3.2",
+    "serverless-offline": "^12.0.4",
```

Comment:
Do we need serverless-offline here and in the new cognito-triggers-sls package?

Reply:
Negatory, removing!